packages/application-components/src/components/detail-pages/custom-form-detail-page/custom-form-detail-page.tsx

```diff
@@ -12,11 +12,13 @@ import Spacings from '@commercetools-uikit/spacings';
 import PageHeaderTitle from '../../internals/page-header-title';
 import PageTopBar from '../../internals/page-top-bar';
 import { ContentWrapper, PageWrapper } from '../../internals/page.styles';
+import { appKitDesignTokens } from '../../../theming';
 
 const DetailPageContainer = styled.div`
   background-color: ${customProperties.colorNeutral95};
-  padding: ${customProperties.spacingM};
   border-bottom: 1px ${customProperties.colorNeutral} solid;
+  padding: ${appKitDesignTokens.paddingVerticalForPage}
+    ${appKitDesignTokens.paddingHorizontalForPage};
```

Comment:
Can we have 1 token only?

Reply:
Updated in this commit: 1752af3561e6d0e5cb7d96d2ea8cb37dfb6bfa7d